media/static-bundles.json

```diff
@@ -1494,7 +1484,6 @@
     {
       "files": [
         "protocol/js/protocol-modal.js",
-        "js/base/mozilla-lazy-load.js",
         "js/foundation/annual-report-2019-2020.es6.js"
```

Comment:
Let's remove the `window.Mozilla.LazyLoad.init();` from this JS file since I'm getting console errors when loading one of the SOM pages